### PR TITLE
[ADD] runbot_website_display

### PR DESCRIPTION
**Pull request by hbrunn**

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Please call it _website_runbot_display_

**Comment by hbrunn** [comment]
it's a specification of runbot, so I think it should start with `runbot_`

**Comment by pedrobaeza** [comment]
But the modification is on the frontend part, so first name should be website. See for example website_sale, website_event, website_crm... they are all about other areas, but on the frontend, so they're prefixed with this word.

File: runbot_website_display/README.rst [hunk 1]
```diff
@@ -0,0 +1,61 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+======================
+Runbot display options
+======================
+
+This module was written to allow users to reorder runbot branches and to hide
```
Comment (elicoidal) [inline runbot_website_display/README.rst]: @hbrunn Thanks for the contribution
"This module allows users to reorder runbot branches and to hide uninteresting ones."

File: runbot_website_display/README.rst [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+======================
+
+This module was written to allow users to reorder runbot branches and to hide 
+uninteresting ones.
+
+Configuration
+=============
+
+After installation, a runbot repository has a checkbox `website_published` you
+can turn off if you don't want to see a repository on the website.
```
Comment (elicoidal) [inline runbot_website_display/README.rst]: After installation, a runbot repository has a checkbox `website_published` that you
can turn off if you don't want to see the repository on the website.

File: runbot_website_display/README.rst [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+   :alt: Odoo Community Association
+   :target: https://odoo-community.org
+
+This module is maintained by the OCA.
+
+OCA, or the Odoo Community Association, is a nonprofit organization whose
+mission is to support the collaborative development of Odoo features and
+promote its widespread use.
+
+To contribute to this module, please visit http://odoo-community.org.
```
Comment (elicoidal) [inline runbot_website_display/README.rst]: https

File: runbot_website_display/__init__.py [hunk 4]
```diff
@@ -0,0 +1,5 @@
+# -*- coding: utf-8 -*-
+# © 2015 Therp BV <http://therp.nl>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from . import models
```
Comment (elicoidal) [inline runbot_website_display/__init__.py]: Please sort all imports in alphabetical order following: https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#imports

File: runbot_website_display/__openerp__.py [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+    "summary": "Allows to reorder and hide runbot repositories",
+    "depends": [
+        'runbot',
+    ],
+    "data": [
+        "views/runbot_repo.xml",
+    ],
+    "images": [
+    ],
+    "pre_init_hook": False,
```
Comment (elicoidal) [inline runbot_website_display/__openerp__.py]: Is it necessary to is all keys set as false?
Please remove empty keys

File: runbot_website_display/models/runbot_repo.py [hunk 6]
```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*-
+# © 2015 Therp BV <http://therp.nl>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openerp import api, fields, models
```
Comment (NL66278) [inline runbot_website_display/models/runbot_repo.py]: Remove api. Not used, leading to F401 error in flake.

**Comment by NL66278** [comment]
@hbrunn  @pedrobaeza Very usefull module. Did some user testing and code review. When name discussion resolved and minor stuff fixed ready to merge. One thing: travis fails because runbot module is missing. This is of course in the huge extra addons repo of Odoo SA. Is there not a case to put this module to this repository, if need be with some synchronization  with the original? Otherwise somebody with the powers to do that should add the extra addons repo to the travis environment.

**Comment by pedrobaeza** [comment]
You can add that repository for Travis declaring it in oca_dependencies.txt file.

**Comment by hbrunn** [comment]
@pedrobaeza @NL66278 I already made the necessary changes in https://github.com/OCA/runbot-addons/pull/73/files's `.travis.yml`

@pedrobaeza `website_sale` et al are modules that bring some backend functionality to the website, so `website_` makes sense here. Runbot intrinsically has a website frontend, so prepending a module that changes some of that with `website_` makes just as much sense to me as prepending a module changing a backend form with `web_`.

**Comment by pedrobaeza** [comment]
@hbrunn, my mental scheme leads me to the contrary, but I won't block the PR by the name.

**Comment by jjscarafia** [comment]
@pedrobaeza I agry with @hbrunn on this one about name convention, I've achieved the same conclusion without seeing this PR.
I close my PR #76 because it does exactly the same as this one.  The only difference is that I inherit two q-web views to limit the sort and @hbrunn inherits the controller.
:+1: for me!

**Comment by nhomar** [comment]
+1 to the @hbrunn name.
+1 to the devel.

**Comment by OSguard** [comment]
@hbrunn please rebase

**Comment by hbrunn** [comment]
@OSguard done

**Comment by OSguard** [comment]
CI tells:

Error details:
Element '<field name="mode">' cannot be located in parent view
Error context:
View `runbot.repo form`
[view_id: 368, xml_id: n/a, model: runbot.repo, parent_id: 327]" while parsing /home/travis/build/OCA/runbot-addons/runbot_website_display/views/runbot_repo.xml:4, near

**Comment by hbrunn** [comment]
@OSguard thanks, now it should work as well with 8.0 and with master

**Comment by tremlin** [comment]
After code review and brief testing: :+1: for merging.

**Comment by OSguard** [comment]
review, no test
:+1:

**Pull request merged**